faker.go

```diff
@@ -4,21 +4,27 @@ package faker
 // Save your time, and Fake your data for your testing now.
 import (
 	"errors"
+	"fmt"
 	"math/rand"
-	"net"
 	"reflect"
+	"sync"
 	"time"
 )
 
 var src = rand.NewSource(time.Now().UnixNano())
+var mu sync.Mutex
```

Comment:
you mutex is still nil. This can cause nil pointer  dereference

Reply:
got any suggestions?
I can insert in every function constructor and init

Reply:
Maybe you just define like `src` 

`mu = sync.Mutex{}`
as the package variable

Reply:
do not understan...
maybe there is an example?

Reply:
var mutex = &sync.Mutex{}

Reply:
aaa now I understand

Reply:
done